api_payments.yaml

```diff
@@ -0,0 +1,286 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy PagoPA
+  description: Proxy for PagoPA payment services
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  "/payment/check":
+    x-swagger-router-controller: PaymentController
+    get:
+      operationId: checkPaymentToPagoPa
```

Comment:
probably, as we are into Proxy PagoPa Api, the suffix `ToPagoPa` is redundant (`checkPayment` suffices imho)